### PR TITLE
release: v1.11.1

### PR DESCRIPTION
**Pull request by jeasonstudio**
Packages published successfully:
🦋  @ant-design/web3-assets@1.7.1
🦋  @ant-design/web3-bitcoin@1.1.0
🦋  @ant-design/web3-common@1.9.1
🦋  @ant-design/web3-eth-web3js@1.0.0
🦋  @ant-design/web3-ethers@1.0.2
🦋  @ant-design/web3-ethers-v5@1.0.0
🦋  @ant-design/web3-solana@1.1.2
🦋  @ant-design/web3-wagmi@2.4.4
🦋  @ant-design/web3@1.11.1

### CODE REVIEW

File: examples/eth-web3js/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# @example/eth-web3js
+
+## null
```
Comment (kiner-tang) [inline examples/eth-web3js/CHANGELOG.md:3]: 奇怪，咋又有 null
```suggestion
```

File: examples/eth-web3js/package.json [hunk 2]
```diff
@@ -22,5 +22,6 @@
     "@types/react": "^18.2.73",
     "@types/react-dom": "^18.2.22",
     "typescript": "^5.4.3"
-  }
+  },
+  "version": null
```
Comment (kiner-tang) [inline examples/eth-web3js/package.json:26]: 这个为啥设置为 null 呀

File: examples/ethers-v5/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,20 @@
...
+- Updated dependencies [17c450f]
+- Updated dependencies [1270c95]
+- Updated dependencies [02e74aa]
+- Updated dependencies [1270c95]
+  - @ant-design/web3@1.11.1
+  - @ant-design/web3-assets@1.7.1
+  - @ant-design/web3-common@1.9.1
+  - @ant-design/web3-ethers-v5@1.0.0
+
+## null
```
Comment (kiner-tang) [inline examples/ethers-v5/CHANGELOG.md:17]:
```suggestion
```

File: examples/ethers/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,19 @@
...
+- Updated dependencies [8c59ee1]
+- Updated dependencies [17c450f]
+- Updated dependencies [1270c95]
+- Updated dependencies [02e74aa]
+  - @ant-design/web3@1.11.1
+  - @ant-design/web3-ethers@1.0.2
+  - @ant-design/web3-assets@1.7.1
+  - @ant-design/web3-common@1.9.1
+
+## null
```
Comment (kiner-tang) [inline examples/ethers/CHANGELOG.md:16]:
```suggestion
```

File: examples/eth-web3js/CHANGELOG.md (hunk 1 above)
Comment (jeasonstudio) [reply examples/eth-web3js/CHANGELOG.md:3]: 这个 null 好像是 pnpm release 的时候自动加的，按理说 private: true 应该完全忽略才对啊

**Review by thinkasany — APPROVED** [review]

**Review by kiner-tang — APPROVED** [review]

**Pull request merged**